src/main/groovy/net/wasdev/wlp/gradle/plugins/utils/LooseApplication.groovy

```diff
@@ -36,8 +36,8 @@ public class LooseApplication {
         config.addDir(parent, task.getProject().getAbsolutePath(), target);
     }
 
-    public void addOutputDirectory(Element parent, Task task, String target) {
-        config.addDir(parent, task.getProject().sourceSets.main.getOutput().getClassesDirs().getSingleFile().getAbsolutePath(), target);
+    public void addOutputDirectory(Element parent, Project project, String target) {
```

Comment:
I think it would be better to add the new method and have the old method to call the new.

Reply:
never mind on this comment.